.pre-commit-config.yaml

```diff
@@ -11,21 +11,16 @@ repos:
       - id: check-executables-have-shebangs
       - id: check-shebang-scripts-are-executable
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.11.0
+    rev: v0.11.4
     hooks:
       - id: ruff
         args: [--fix]
       - id: ruff-format
-  - repo: local
+  - repo: https://github.com/shellcheck-py/shellcheck-py
```

Comment:
Out of curiosity, was there an issue with the "official" one? Was it not to depend on docker? 

https://github.com/koalaman/shellcheck-precommit

Reply:
I see, thank you!

Reply:
The docker stuff was also the reason why I did not use the official one... So my fallback was the local installation.